docs/snapshotting/snapshot-support.md

```diff
@@ -668,9 +665,10 @@ m6i won't work on an m6g instance.
 | *CPU family* | *taken on host kernel* | *restored on host kernel* | *working?* |
 | ------------ | ---------------------- | ------------------------- | ---------- |
 | **x86_64**   | 4.14                   | 5.10                      | Y          |
-| **all**      | 5.10                   | 4.14                      | N          |
-| **all**      | 5.10                   | 6.1                       | Y          |
-| **all**      | 6.1                    | 5.10                      | Y          |
+| **x86_64**   | 5.10                   | 6.1                       | Y          |
```

Comment:
a user may ask "since 4.14 -> 5.10 and 5.10 -> 6.1 work, does 4.14 -> 6.1 also work?". Would we like to mention that explicitly or is it inferred from the (implicit) transitivity?

Reply:
Hmm true we don't explicitly test it. Maybe we should put sthg to the effect of "we only test one jump between versions"

Reply:
I think we do test it actually: 
![image](https://github.com/firecracker-microvm/firecracker/assets/115481277/3bed6d9c-0f0a-43f3-ba84-105a4ab62666)
:thinking:

Reply:
Will be covered in follow-up